tailwindHyperSwitch.config.js

```diff
@@ -588,28 +588,7 @@ module.exports = {
   variants: {},
   plugins: [
     plugin(function ({ addUtilities }) {
-      const newUtilities = {
-        "*::-webkit-scrollbar": {
-          display: "none", // chrome and other
-        },
-        "*": {
-          scrollbarWidth: "none", // firefox
-        },
-        ".show-scrollbar::-webkit-scrollbar": {
-          display: "block",
-          overflow: "scroll",
-          height: "4px",
-          width: "8px",
-        },
-        ".show-scrollbar::-webkit-scrollbar-thumb": {
-          display: "block",
-          borderRadius: "20rem",
-          backgroundColor: "#8A8C8F",
-        },
-        ".show-scrollbar::-webkit-scrollbar-track": {
-          backgroundColor: "#FFFFFFF",
-        },
-      };
```

Comment:
why this has been removed?

Reply:
We just want to add native scrollbar no other custom changes